library/core/src/result.rs

```diff
@@ -1233,7 +1233,8 @@ impl<T, E> Result<Option<T>, E> {
     /// ```
     #[inline]
     #[stable(feature = "transpose_result", since = "1.33.0")]
-    pub fn transpose(self) -> Option<Result<T, E>> {
+    #[rustc_const_unstable(feature = "const_result2", reason = "newly added", issue = "57563")]
```

Comment:
#57563 is the meta tracking issue, it should not be used for any specific feature. If there isn't any tracking issue for const `Result` methods yet, either we need a new one or we could extend the `Option` one to also cover `Result`.

Reply:
I'll update the `Option` ones to #67441, as you noted in the other comment. I don't see a tracking issue for const `Result` functions, so I'll create one in a bit.

Reply:
There is #67520

Reply:
That's fine. I'm creating a relatively thorough one listing all blockers for all methods.

Reply:
Updated to the newly created tracking issue.